### PR TITLE
LG-3313: Pass all Row props to V11Adapter rows

### PR DESCRIPTION
**Pull request by spark33**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-3313](https://jira.mongodb.org/browse/LG-3313)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

**Review by bruugey — CHANGES_REQUESTED** [review]
This only solves for row. I believe we need to add this to all elements.

File: packages/table/src/V11Adapter/V11Adapter.story.tsx [hunk 1]
```diff
@@ -52,8 +52,8 @@ export const Basic: StoryFn<StoryAdapterProps> = () => {
         }
       >
         {({ datum }: any) => (
-          <Row>
-            <Cell>{datum.name}</Cell>
+          <Row data-test="test-row">
```
Comment (bruugey) [inline packages/table/src/V11Adapter/V11Adapter.story.tsx:55]: confirmed this works, we can clean up storybook.

File: packages/table/src/V11Adapter/V11Adapter.tsx [hunk 2]
```diff
@@ -51,8 +52,12 @@ const V11Adapter = <T extends LGRowData>({
   const containerRef = useRef(null);
   const OldTable = flattenChildren(children)[0];
 
+  consoleOnce.warn(
```
Comment (bruugey) [inline packages/table/src/V11Adapter/V11Adapter.tsx:55]: are there things we should be explicitly filtering out?

File: packages/table/src/V11Adapter/V11Adapter.tsx [hunk 3]
```diff
@@ -142,9 +169,15 @@ const V11Adapter = <T extends LGRowData>({
                       // index by row.index (not the index of the loop) to get the sorted order
                       // @ts-expect-error `processedData` is structured to be indexable by `row.index`
                       processedData[row.index]?.[cell.column.id]?.();
+                    const {
+                      children,
+                      isHeader,
```
Comment (bruugey) [inline packages/table/src/V11Adapter/V11Adapter.tsx:174]: Are `isHeader` and `isDisabled` the only props that need filtering out?

Comment (spark33) [reply packages/table/src/V11Adapter/V11Adapter.tsx:174]: image

**Review by bruugey — COMMENTED** [review]
we need a changeset here

File: packages/table/src/V11Adapter/V11Adapter.tsx [hunk 4]
```diff
@@ -47,17 +48,17 @@ const V11Adapter = <T extends LGRowData>({
...
   const OldTable = flattenChildren(children)[0];
 
   if (!isComponentType(OldTable, 'Table')) {
-    console.error(
+    consoleOnce.error(
       'The first and only child of `Table.V11Adapter` must be a `V10Table` component',
     );
   }
 
   const OldTableProps = (OldTable as ReactElement).props;
```
Comment (TheSonOfThomp) [inline packages/table/src/V11Adapter/V11Adapter.tsx:60]: Doesn't `isComponentType` assert that the variable is the right type?

**Review by TheSonOfThomp — APPROVED** [review]
LGTM. Tests seems to verify the behavior we want

**Review by bruugey — APPROVED** [review]

**Pull request closed without merging**